tokens/global/space.json5

```diff
@@ -0,0 +1,202 @@
+{
+  space: {
+    'eighth-x': {
+      value: '2px',
```

Comment:
Value should be unitless, but we can add something to strip the 'px' pretty easily if we want

Reply:
I feel we should try to be agnostic in the source, so unit-less makes the most sense to me.

Reply:
thats fine with me as long as we are all working from the point of these "being" px values as rem em and dp will all be wildly inaccurate

Reply:
It's pretty easy to strip the px